src/responses.rs

```diff
@@ -1,11 +1,13 @@
+use std::io;
 use rocket::serde::{Deserialize, Serialize};
-use rocket::serde::json::{Json};
-use rocket::http::{ContentType};
+use rocket::serde::json::{Json, json};
+use rocket::http::{ContentType, Status};
+use rocket::{Request, Response};
 use rocket::response::Responder;
 
 #[derive(Responder)]
 #[response(status = 200, content_type = "json")]
-pub struct SuccessResponder<T> {
+pub struct OutpackSuccess<T> {
```

Comment:
renamed for symmetry with `OutpackError`, which now also implements the `rocket::Responder` trait (which all endpoints need to return)